spec/open_api_spec.rb

```diff
@@ -89,10 +90,44 @@ def random_path
     end
   end
 
+  describe "Openapi schema attributes" do
+    PARSER_ATTRS = %w[readOnly nullable enum].freeze
+    context "correctly configured" do
+      ActiveRecord::Base.connection.tables.each do |table|
+        next if SKIP_TABLES.include?(table)
+
+        model_name = table.singularize.camelize
+        doc = Api::Docs["1.0"].definitions
```

Comment:
Does this "1.0" need to change with each API version change?  If so, will we want to check all versions once we support multiple?  Will this need to be yet another loop over each version?

Reply:
Building out the validation structure that mimics the openapi_parser may help with the lower level recursion, but I'm thinking we would need to loop over every version we support in the future.